src/etl/do_etl.py

```diff
@@ -180,75 +204,79 @@ def get_generators(self, filepath, batch_size):
+            all_parents_subont = ont.subontology(all_parents)
+            isas_without_names = all_parents_subont.parents(key, relations=['subClassOf'])
 
-            for item in isasWithoutNames:
+            for item in isas_without_names:
                 dictionary = {
-                    "primary_id": k,
+                    "primary_id": key,
                     "primary_id2": item
                 }
 
                 do_isas_list.append(dictionary)
 
-            defLinksProcessed = []
-            defLinks = ""
+            def_links_processed = []
+            def_links = ""
             if definition is None:
                 definition = ""
             else:
-                #definition = definition.replace('\n', ' ') # Remove new lines that cause this to split across two lines in the file
-                #definition = definition.replace('  ', ' ') # Remove any extra double space that might have been introduces in the last replace
+                # Remove new lines that cause this to split across two lines in the file
+                # definition = definition.replace('\n', ' ')
+
+                # Remove any extra double space that might have been introduces in the last replace
+                # definition = definition.replace('  ', ' ')
+
                 if definition is not None and "\"" in definition:
                     split_definition = re.split(r'(?<!\\)"', definition)
                     if len(split_definition) > 1:
                         if len(split_definition) > 2 and "[" in split_definition[2].strip():
-                            defLinks = split_definition[2].strip()
-                            defLinks = defLinks.rstrip("]").replace("[", "")
-                            defLinksUnprocessed.append(defLinks)
-
-
-
-            for dl in defLinksUnprocessed:
-                dl = dl.replace("url:www", "http://www")
-                dl = dl.replace("url:", "")
-                dl = dl.replace("URL:", "")
-                dl = dl.replace("\\:", ":")
-                dl = dl.replace('\\', '')
-
-                if "," in dl:
-                    dl = dl.split(",")
-                    for link in dl:
+                            def_links = split_definition[2].strip()
+                            def_links = def_links.rstrip("]").replace("[", "")
+                            def_links_unprocessed.append(def_links)
+
+            for def_link in def_links_unprocessed:
+                def_link = def_link.replace("url:www", "http://www")
+                def_link = def_link.replace("url:", "")
+                def_link = def_link.replace("URL:", "")
+                def_link = def_link.replace("\\:", ":")
+                def_link = def_link.replace('\\', '')
+
+                if "," in def_link:
+                    def_link = def_link.split(",")
+                    for link in def_link:
                         if link.strip().startswith("http"):
-                            defLinksProcessed.append(link)
```

Comment:
removing commented out code is probably better than reformatting it.

Reply:
I didn't delete commented out code because I was not certain if it was left in the code for a reason. I will delete this code as you pointed this out and It doesn't look like it is code that would be hard to recreate. If you see other commented out code that you think should go please feel free to point it out and I will delete it.